os/osx.cmake

```diff
@@ -52,9 +52,22 @@ if(_is_empty)
   endif()
 endif()
 
+# With a full Xcode install, typically `xcode-select -print-path` is something like:
+#     /Applications/Xcode.app/Contents/Developer
+#
+# But with just Xcode command line tools installed, the path is:
+#     /Library/Developer/CommandLineTools
+#
+# In the CommandLineTools case, the SDKs folder is at a different relative path.
+if(XCODE_DEVELOPER_ROOT MATCHES ".*CommandLineTools.*")
```

Comment:
I think this condition is not very robust. What about checking if `"${XCODE_DEVELOPER_ROOT}/Platforms/MacOSX.platform/Developer/SDKs/MacOSX${OSX_SDK_VERSION}.sdk"` path exists and if it's not exist try `"${XCODE_DEVELOPER_ROOT}/SDKs/MacOSX${OSX_SDK_VERSION}.sdk"`?

Reply:
> Works for me, thanks.

Does it mean you will update this pull request?

Reply:
Yes, will do